packages/connect-examples/expo-playground/app/constants/connect.ts

```
@@ -1,4 +1,4 @@
-import { getConnectSrc } from '../../../constants';
+import { getConnectSrc } from '@onekey-internal/shared-constants';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)